### ISSUE TITLE
Turn off "Unknown Command"?

### ISSUE THREAD

**Issue by Aida-Enna** [RogueException/Discord.Net#1090]
Is there any way to turn off the bot responding to non-valid commands with "Unknown command"? We've had a bot in our server for a while and it uses "~" for it's start, but when people make faces like "~owo~" or "~12 of these items", the bot responds with "Unknown Command".

I'd like to disable this - Is that possible?

**Comment by PassiveModding** [comment]
talk to the bot owner. That is not a Discord.Net issue

**Comment by Still34** [comment]
Assuming this bot is yours, this behavior of replying to user about the status of the command execution is defined within your command handler. By monitoring the result from `CommandService.ExecuteAsync` you can already determine whether or not to reply to the user according to the type of error that it may have encountered via [IResult.Error](https://docs.stillu.cc/api/Discord.Commands.IResult.html#Discord_Commands_IResult_Error)

In other words, change your behavior defined after the execution and see if the error matches `CommandError.UnknownCommand`, if so, skip it.

**Comment by Aida-Enna** [comment]
@PassiveModding I am the bot owner, hence my asking.

@Still34 Awesome, I'll do that. Thanks! :)

**Comment by Still34** [comment]
Additional note, if it's a general question rather than a library-related erroneous issue, consider visiting us on `#dotnet_discord-net`.
